platform/security/pdp/security-pdp-authzrealm/src/main/java/ddf/security/pdp/realm/AuthzRealm.java

```diff
@@ -504,4 +506,9 @@ public void setMatchOneMappings(List<String> list) {
             }
         }
     }
+
+    public void setEnvironmentAttributes(List<String> environmentAttributes) {
+        this.environmentAttributes.clear();
+        this.environmentAttributes.addAll(environmentAttributes);
```

Comment:
Are we okay with this not being thread safe, its an edge case for sure but we could just use a new ArrayList and replace environmentAttributes